docs/README.md

```diff
@@ -420,93 +420,186 @@ Depending on being a Data Provider or a Data Consumer there are different tokens
 #### Data Consumer
 1. needs an EDR token which is provided between the intercommunication between the EDCs.
 
+### Authentication on behalf of a user
+The AAS Registry can be accessed on behalf of a user. The token has to be obtained via the OpenID Connect flow. The AAS Registry will validate these tokens.
+
+#### Postman configuration
+![](img/image005.png)
+
+*Support contact*	tractusx-dev@eclipse.org
+
+### Access control to Digital Twins Based on the BPN (Business Partner Number)/ TenantId
+The visibility of `specificAssetIds` in the Digital Twin Registry based on the Business Partner Number (BPN) can be controlled with the attribute `externalSubjectId`. Hence, the `externalSubjectId` is identified with the BPN.
+
+* The BPN as attribute to a *specificAssetId* can be optionally provided in `specificAssetIds`. This can be done with `externalSubjectId`.
+* Only those users, where `externalSubjectId` matches the Eclipse Dataspace Components-Header (*i.e.* BPN) are able to discover via and read exactly this content of `specificAssetIds`.
```

Comment:
Wouldn't make it more sense to to write this sentence "...  are able to discover and read ..."? So without "via"?